src/stream/src/executor/hash_join.rs

```diff
@@ -659,6 +659,11 @@ impl<K: HashKey, S: StateStore, const T: JoinTypePrimitive> HashJoinExecutor<K,
                         self.chunk_size,
                     ) {
                         yield chunk.map(|v| match v {
+                            Message::Watermark(_) => {
+                                unimplemented!(
```

Comment:
```suggestion
                                todo!(
```

Same for other occurrences.